### ISSUE TITLE
Race condition when binding to live reload port

### ISSUE THREAD

**Issue by paddor** [ember-cli/ember-cli#7509]
Hi! Moving this here from emberjs/ember.js#15975. There seems to be a race condition between finding a free port for the live reload server and binding to it.

I noticed this problem  when starting two `ember serve` processes to test a high availability setup using a Procfile and Hivemind/Overmind (they're like Foreman), one of them usually fails like this:

```
Livereload failed on http://localhost:49153.  It is either in use or you do not have permission.
```

Relevant lines from the Procfile:
```
ember-primary: cd webui && ember serve --port 4200
ember-backup: cd webui && ember serve --port 4201
```

Running one of them with a prepended `sleep 10;` often helps, but not always. I guess making that section atomic might be difficult, but at least it could try to find another free port if the first one turns out to be in use?

Thanks.

Versions:
```
$ ember version --verbose
ember-cli: 2.15.1
http_parser: 2.7.0
node: 7.10.0
v8: 5.5.372.43
uv: 1.11.0
zlib: 1.2.11
ares: 1.10.1-DEV
modules: 51
openssl: 1.0.2k
icu: 58.2
unicode: 9.0
cldr: 30.0.3
tz: 2016j
os: linux x64
$ npm --version
4.2.0
```

**Comment by rwjblue** [comment]
As a work around, you can provide your love reload port numbers to avoid the auto-detection race condition...

**Comment by stefanpenner** [comment]
We are aware of this race-condition, I believe the solution is to multi-plex LR over the same port as HTTP which is something @chrmod has begun https://github.com/ember-cli/ember-cli/pull/7508

**Comment by rwjblue** [comment]
I am excited for that!  However, it doesn't really resolve the race condition situation in general...

**Comment by paddor** [comment]
@rwjblue If there's no reload port needed, only the normal (fixed) port alone is needed. I'd be happy to see that change.

**Comment by stefanpenner** [comment]
Fixed by https://github.com/ember-cli/ember-cli/pull/7940 which is available in 3.4.0-beta.2
